qa/suites/fs/upgrade/mds_upgrade_sequence/overrides/ignorelist_upgrade.yaml

```diff
@@ -2,3 +2,4 @@ overrides:
   ceph:
     log-ignorelist:
       - OSD_DOWN
+      - 'Failed to load ceph-mgr modules: k8sevents'
```

Comment:
Not sure we actually want to ignore this one. This was introduced by the centos 9 container changes and is something we've actually been trying to fix.